### PR TITLE
Remove webrtc support

### PR DESCRIPTION
**Pull request by joehand**
Mostly just doc updates.

Do we want to include this line? 

```js
if (args.webrtc) onerror('webrtc no longer supported')
```

### CODE REVIEW

**Comment by karissa** [comment]
yeah, and maybe including a link to an explanation if possible (in the faq?)

**Comment by joehand** [comment]
> yeah, and maybe including a link to an explanation if possible (in the faq?)

👍 . I can do that this afternoon. I felt like the docs needed a better explanation than [what I put too](https://github.com/datproject/dat/pull/572/files#diff-04c6e90faac2675aa89e2176d2eec7d8R196).

File: bin/cli.js [hunk 1]
```diff
@@ -9,8 +9,7 @@ var Dat = require('dat-js')
 
 var args = require('minimist')(process.argv.splice(2), {
   alias: {p: 'port', q: 'quiet', v: 'version'},
-  boolean: ['watchFiles', 'snapshot', 'exit', 'list', 'quiet', 'version', 'utp', 'temp', 'webrtc', 'ignore-hidden'],
-  string: ['signalhub'],
```
Comment (juliangruber) [inline bin/cli.js]: why did you remove the `"signalhub"` option too?

Comment (joehand) [inline bin/cli.js]: @juliangruber signalhub is only used for webrtc.

Comment (juliangruber) [inline bin/cli.js]: duh :D ok, all +1 then
